File: ts-sdk/src/sdk/models/operations/listconfigsavailableconnectors.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";


export class ListConfigsAvailableConnectorsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  connectorsConfigsResponse?: shared.ConnectorsConfigsResponse;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-sdk/pkg/models/operations/listconfigsavailableconnectors.go
package operations

import (
	"github.com/speakeasy-sdks/formance-sdks/go-client-sdk/pkg/models/shared"
)

type ListConfigsAvailableConnectorsResponse struct {
	ConnectorsConfigsResponse *shared.ConnectorsConfigsResponse
	ContentType               string
	StatusCode                int
}
